components/kyma-operator/cmd/operator/main.go

```diff
@@ -51,6 +56,7 @@ func main() {
 	tlsCrt := flag.String("tillerTLSCrt", "/etc/certs/tls.crt", "Path to TLS cert file")
 	TLSInsecureSkipVerify := flag.Bool("tillerTLSInsecureSkipVerify", false, "Disable verification of Tiller TLS cert")
 	backoffIntervalsRaw := flag.String("backoffIntervals", "10,20,40,60,80", "Number of seconds to wait before subsequent retries")
+	printOverridesToSTDOut := flag.Bool("printOverridesToSTDOut", false, "Print Installation Overrides")
```

Comment:
I suggest to provide a filename as a flag. Default value should be a standard alias /dev/stdout. In this way the Installer behaves in the same way unless you explicitly change the log destination.
Stretch: After some time, when we ensure it works correctly in all environments, we can consider a follow-up task that changes the default to /dev/null. Then you **have to** explicitly tell the Installer where to log this stuf.